gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/store/setLoweringAccessController.ts

```diff
@@ -0,0 +1,62 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { PublicKey } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '../../../lib/contracts'
+
+export default class SetAccessController extends SolanaCommand {
+  static id = 'store:set_lowering_access_controller'
+  static category = CONTRACT_LIST.STORE
+
+  static examples = [
+    'yarn gauntlet store:set_lowering_access_controller --network=local --state=EPRYwrb1Dwi8VT5SutS4vYNdF8HqvE7QwvqeCCwHdVLC --accessController=EPRYwrb1Dwi8VT5SutS4vYNdF8HqvE7QwvqeCCwHdVLC',
```

Comment:
Same comment as above, as I don't really find these examples readable or the provided data useful.